### PR TITLE
improve some wording

### PR DESCRIPTION
**Pull request by marcoow**
some more improvements

### CODE REVIEW

File: _includes/blog_header.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <div class="row">
       <div class="col-sm-6 col-sm-offset-3">
         <h2>Blog</h2>
-        <p>We write about <strong>Ember.js</strong>, <strong>Ruby on Rails</strong> as well as <strong>Elixir</strong> and <strong>Phoenix</strong>.</p>
+        <p>We write about Web Engineering, <strong>Ember.js</strong>, <strong>Ruby on Rails</strong> as well as <strong>Elixir</strong> and <strong>Phoenix</strong>.</p>
```
Comment (oliverbarnes) [inline _includes/blog_header.html]: I think the same thing I've been noting about capitalized common nouns applies here. But it'd be good to see other examples of this, do you have any references?

File: services/index.html [hunk 2]
```diff
@@ -22,7 +22,7 @@
           <h3>Software Engineering</h3>
         </div>
         <div class="col-sm-6">
-          <p>simplabs builds solutions for the web and mobile devices for everyone from small startups to big enterprises. Our solutions are <strong>well architected and easy to maintain and extend</strong> by our clients once we're gone.</p>
+          <p>simplabs builds solutions for the web and mobile devices for everyone from small startups to big enterprises. Our solutions are <strong>well architected and easy to maintain and extend</strong> by our clients once we have left.</p>
```
Comment (oliverbarnes) [inline services/index.html:25]: if we're worried about "gone" and "left", we can also say "once our engagement ends" or something like it

**Review by oliverbarnes — CHANGES_REQUESTED** [review]

File: _includes/blog_header.html (hunk 1 above)
Comment (marcoow) [reply _includes/blog_header.html]: Hm, no idea really. If you say lowercase I'll lowercase it.

File: services/index.html (hunk 2 above)
Comment (marcoow) [reply services/index.html:25]: I think I had changed _"we're gone"_ to _"we have left"_ at a few other places and simply forgot this one so this is just aligning this paragraph with the other text.

File: _includes/blog_header.html (hunk 1 above)
Comment (oliverbarnes) [reply _includes/blog_header.html]: lowercase it then :)

File: services/index.html (hunk 2 above)
Comment (oliverbarnes) [reply services/index.html:25]: 👍

File: _includes/blog_header.html (hunk 1 above)
Comment (marcoow) [reply _includes/blog_header.html]: fixed

**Review by oliverbarnes — APPROVED** [review]

**Pull request merged**